### ISSUE TITLE
./node_modules/ag-grid-community/styles/ag-grid.css ParserError: Syntax Error at line: 1, column 83

### ISSUE THREAD

**Issue by mkinare** [ag-grid/ag-grid#5580]
**I'm submitting a,** 
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Description**

I have been trying to build react code to deploy in prod and with ag-grid-community@28.1.1. Dev environment works as expected, however during build process I encounter an error.

**Current behavior**
```
react-app-rewired build

Creating an optimized production build...
Failed to compile.

./node_modules/ag-grid-community/styles/ag-grid.css
ParserError: Syntax Error at line: 1, column 83
```

**Expected behavior**

Should be able to build the code for production.

**AG Grid version:** 28.1.1

**package.json**

{
  "name": "inhouseapp",
  "version": "5.1.0",
  "private": true,
  "dependencies": {
    "@babel/plugin-proposal-decorators": "^7.12.1",
    "@blueprintjs/core": "^4.5.1",
    "@blueprintjs/select": "^4.4.1",
    "@blueprintjs/table": "^4.3.1",
    "@devexpress/dx-react-core": "^3.0.5",
    "@devexpress/dx-react-grid": "^3.0.5",
    "@devexpress/dx-react-grid-material-ui": "^3.0.5",
    "@emotion/react": "^11.10.0",
    "@emotion/styled": "^11.10.0",
    "@material-table/core": "4.3.39",
    "@material-table/exporters": "^1.0.12",
    "@material-ui/core": "^4.12.3",
    "@mui/icons-material": "^5.0.1",
    "@mui/lab": "^5.0.0-alpha.85",
    "@mui/material": "^5.10.2",
    "@mui/x-date-pickers": "^5.0.0-beta.5",
    "@testing-library/jest-dom": "^5.14.1",
    "@testing-library/react": "^11.2.7",
    "@testing-library/user-event": "^12.8.3",
    "ag-grid-community": "^28.1.1",
    "ag-grid-react": "^28.1.1",
    "blob-util": "^2.0.2",
    "bootstrap": "^5.0.2",
    "core-js": "^3.18.0",
    "customize-cra": "^1.0.0",
    "material-table": "^1.69.3",
    "normalize.css": "^8.0.1",
    "plotly.js": "^2.3.0",
    "react": "^17.0.2",
    "react-app-rewired": "^2.2.1",
    "react-dom": "^17.0.2",
    "react-idle-timer": "^4.6.4",
    "react-json-to-csv": "^1.0.4",
    "react-loader-spinner": "^4.0.0",
    "react-plotly.js": "^2.5.1",
    "react-router-dom": "^5.2.0",
    "react-scripts": "4.0.3",
    "react-select": "^4.3.1",
    "react-spring": "^9.4.5",
    "react-tracking": "^9.0.0",
    "uuid": "^8.3.2",
    "web-vitals": "^1.1.2"
  },
  "scripts": {
    "start": "node ./config-overrides.js && cross-env BROWSER='chrome' react-app-rewired start",
    "build": "react-app-rewired build",
    "rmod": "remove_modules.bat",
    "test": "react-app-rewired test",
    "eject": "react-app-rewired eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "cross-env": "^7.0.3"
  }
}

**Known difference from CRA**

I am using react-app-rewired so that I can use @ decorators. 

**What is in the file ./node_modules/ag-grid-community/styles/ag-grid.css**

Nothing is on line 1, column 83
```
ag-icon {
  font-family: var(--ag-icon-font-family);
  font-size: var(--ag-icon-size);
  line-height: var(--ag-icon-size);
  font-style: normal;
  font-weight: normal;
  font-variant: normal;
  text-transform: none;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

.ag-icon-aggregation::before {
  content: var(--ag-icon-font-code-aggregation, "\f101");
}

.ag-icon-arrows::before {
  content: var(--ag-icon-font-code-arrows, "\f102");
}

.ag-icon-asc::before {.....
```

**Comment by ellyse18** [comment]
I had the same issue. In the end I was able to resolve it by updating postcss-values-parser in npm. Previously it was v2.01 which couldn't process the .css file. Did not need the workaround as provided by @mkinare but your bug report gave me the guidance to resolve this. Thanks.

**Comment by StephenCooper** [comment]
**AG-7222 Add Post Processing of CSS for CRA 3.x/4.x Compatibility** which should help address this issue will be released as part of our next release 28.2 and should resolve these issues without the need for a workaround.
